pkg/utils/network.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"strings"
+)
+
+func SplitCIDR(s string) (string, string, error) {
+	split := strings.Split(s, "/")
+	if len(split) != 2 {
+		return "", "", fmt.Errorf("not a CIDR")
+	}
+
+	return split[0], split[1], nil
+}
+
+func IsIPv4Addr(ip string) bool {
+	return strings.Contains(ip, ".") && net.ParseIP(ip) != nil
+}
+
+func IsIPv6Addr(ip string) bool {
```

Comment:
Note that we already have an IsIPv6 function: https://github.com/openshift/baremetal-runtimecfg/blob/2905c0425f6e8f2952d366b130f5583da04e1e6d/pkg/utils/addresses.go#L131

Reply:
LOL of course I see it now. I will clean&fix this

Reply:
The current implementation,
```
func IsIPv6(ip net.IP) bool {
	return ip.To4() == nil
}
```
is prone to give wrong results when running against malformed IPs. I will let myself to change it, while keeping the same signature